src/histograms/utils/HistogramUtils.ts

```diff
@@ -403,3 +404,79 @@ export enum ChartType {
 export enum Position {
   top, bottom
 }
+
+
+export const SELECTED_STYLE: Style = {
+  fill: '#5CCBC3',
+  stroke: '#5CCBC3',
+  stroke_width: 1,
+  background_color: '#FFF',
+  background_opacity: 0.2
+};
+
+export const UNSELECTED_STYLE: Style = {
+  fill: '#DADADA',
+  stroke: '#DADADA',
+  stroke_width: 1,
+  background_color: '#FFF',
+  background_opacity: 0
+};
+
+export const HEAD_BAR = {
+  SELECTED_STYLE,
+  UNSELECTED_STYLE,
+  HEIGHT: 5
+};
+
+export const BAR_OPTIONS = {
+  WEIGHT: 0.8,
+  BACKGROUND_COLOR: '#FFF'
+};
+
+export function getBarOptions(barOptions: BarOptions): BarOptions {
+  const returnedBarOptions: BarOptions = barOptions ? Object.assign({}, barOptions) : {};
```

Comment:
I don't understand why you need to assign barOptions to an empty object

Reply:
To clone it and keep it immutable. `returnedBarOptions` on the other hand take default values if a parameter is not defined